lib/Frontend/PrintingDiagnosticConsumer.cpp

```diff
@@ -371,6 +371,7 @@ static void enqueueDiagnostic(
 ///
 /// The resulting vector will always contain valid source locations. If the
 /// initial location is invalid, the result will be empty.
+#if SWIFT_BUILD_SWIFT_SYNTAX
```

Comment:
@etcwilde fixed this on rebranch - would you mind changing this to not add the #if here and instead just remove the #if below? ie. making all these 3 functions in the same `#if SWIFT_BUILD_SWIFT_SYNTAX` branch?